ws-nextjs-app/pages/[service]/send/[id]/FormField/FieldLabel.tsx

```diff
@@ -5,15 +5,18 @@ import Text from '#app/components/Text';
 import { InputProps } from '../types';
 import styles from './styles';
 
+const translation = '(optional)';
```

Comment:
Should we rename this to indicate that its the 'optional' translation. Temporary I guess, since we'll be getting the translations from Context anyway.